### PR TITLE
MP-594

### PR DESCRIPTION
**Pull request by LuudSlagter**

### CODE REVIEW

File: SearchParameters/SearchParameter-period-of-use.xml
Comment (weteringac) [inline SearchParameters/SearchParameter-period-of-use.xml:1]: "even if these are authored outside the PeriodOfUse that is being searched on"

even if these have a period of use outside the ...

Comment (weteringac) [inline SearchParameters/SearchParameter-period-of-use.xml:1]: Ik vind deze uitleg super helder, maar mis daar dat het per MBH moet. Het staat daarboven wel, maar ik zou het hier nogmaals noemen:

For this all MedicationRequest and MedicationDispense resources wherein the [ext-StopType](http://nictiz.nl/fhir/StructureDefinition/ext-StopType) extension is present should be considered, before including (for each pharmaceutical treatment) only the latest one based on the value of `MedicationRequest.authoredOn` and the [ext-AdministrationAgreement.AdministrationAgreementDateTime](http://nictiz.nl/fhir/StructureDefinition/ext-AdministrationAgreement.AdministrationAgreementDateTime) extension, respectively."

**Review by weteringac — CHANGES_REQUESTED** [review]

File: SearchParameters/SearchParameter-period-of-use.xml
Comment (LuudSlagter) [reply SearchParameters/SearchParameter-period-of-use.xml:1]: Fixed!

Comment (LuudSlagter) [reply SearchParameters/SearchParameter-period-of-use.xml:1]: Ik heb '(for each pharmaceutical treatment)' vervangen door ', for each pharmaceutical treatment,' (dit was ook feedback van Jorn).

Comment (weteringac) [reply SearchParameters/SearchParameter-period-of-use.xml:1]: Er staat nog steeds:

even if these are authored outside the PeriodOfUse that is being searched on

Het gaat niet wanneer geregistreerd, maar wanneer het gebruik is.

**Review by weteringac — CHANGES_REQUESTED** [review]

File: SearchParameters/SearchParameter-period-of-use.xml
Comment (LuudSlagter) [reply SearchParameters/SearchParameter-period-of-use.xml:1]: Er staat nu 'even if these have a period of use outside the PeriodOfUse that is being searched on', toch?

**Review by weteringac — APPROVED** [review]

**Pull request merged**